### ISSUE TITLE
MURA

### ISSUE THREAD

**Issue by 3D-ICE-arXiv** [3D-ICE-arXiv/Papers#62]
MURA is a dataset of bone X-rays. The training set contains 36k
X-rays and the validation set contains 3k images

The MURA dataset [14] contains upper limb X-rays
images labeled whether they contain anomaly or not. This dataset is composed
of seven classes of body parts: finger, hand, wrist, forearm, elbow, humerus, and
shoulder. There are a total of 40’005 X-ray images from 11’967 unique patients.
![image](https://user-images.githubusercontent.com/73802219/101691568-36a05900-3ab2-11eb-8984-3117ef3e00b1.png)

Rajpurkar, P., et al.: Mura: Large dataset for abnormality detection in musculoskeletal
radiographs. arXiv preprint arXiv:1712.06957 (2017)

**Comment by 3D-ICE-arXiv** [comment]
https://github.com/3D-ICE-arXiv/Papers/issues/60

**Comment by 3D-ICE-arXiv** [comment]
https://github.com/3D-ICE-arXiv/Papers/issues/70
